src/App.tsx

```diff
@@ -15,71 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
```

Comment:
you'll have bugs if goods array haven't 'Jam'
so i propose to use so
```suggestion
    selectedGood: goods[goods.indexOf('Jam')] || goods[0];,
```

Reply:
ok